### DISCUSSION TITLE
Timepicker ok button not showing when select a time. but when i hover on it the button showing again. how to fix it

### DISCUSSION THREAD

**Discussion by opuopu** [ant-design/ant-design#48880] [Q&A]
hello good people. i use ant design. but the issue is when i select a time the timepicker ok button gone invisible
![image](https://github.com/ant-design/ant-design/assets/43672649/c6fda71e-a12c-469e-baef-1c4b2d37be89)
but when i hover on it by my mouse the background color showing again. 
![image](https://github.com/ant-design/ant-design/assets/43672649/3256adae-180f-4f84-8951-faec6bfe5ff3)
my code:
![image](https://github.com/ant-design/ant-design/assets/43672649/2f49b518-a4c4-44e0-8aa8-23b92574c839)

**Comment by Bbbtt04** [comment]
Can you give a concrete example?

**Reply by opuopu** [reply]
may i know,what do you mean by concrete example?. generally use  use configprovider for change the design of components. then its happend. but i tried lots of time to catch the issue not finding anything.
